schema.sql

```diff
@@ -6,7 +6,23 @@
     escape_attempts INTEGER,
     neutered BOOLEAN,
     weight_kg DECIMAL(10,2));
+  
+ALTER TABLE animals
```

Comment:
Please note that you are supposed to Remove column species and Add column species_id, you already did the latter but instead of add column here you should have drop. 😉